src/renderer/renderElement.ts

```diff
@@ -789,40 +789,45 @@ export function getFreeDrawPath2D(element: ExcalidrawFreeDrawElement) {
 }
 
 export function getFreeDrawSvgPath(element: ExcalidrawFreeDrawElement) {
+  // If input points are empty (should they ever be?) return a dot
   const inputPoints = element.simulatePressure
     ? element.points
     : element.points.length
     ? element.points.map(([x, y], i) => [x, y, element.pressures[i]])
-    : [[0, 0, 0]];
+    : [[0, 0, 0.5]];
 
   // Consider changing the options for simulated pressure vs real pressure
-  const options = {
+  const options: StrokeOptions = {
     simulatePressure: element.simulatePressure,
-    size: element.strokeWidth * 6,
-    thinning: 0.5,
+    size: element.strokeWidth * 4.25,
+    thinning: 0.6,
     smoothing: 0.5,
     streamline: 0.5,
-    easing: (t: number) => t * (2 - t),
-    last: true,
+    easing: (t) => Math.sin((t * Math.PI) / 2),
+    last: false,
   };
 
-  const points = getFreeDrawShape(inputPoints as number[][], options);
-  const d: (string | number)[] = [];
-
-  let [p0, p1] = points;
+  return getSvgPathFromStroke(getStroke(inputPoints as number[][], options));
+}
 
-  d.push("M", p0[0], p0[1], "Q");
+function med(A: number[], B: number[]) {
+  return [(A[0] + B[0]) / 2, (A[1] + B[1]) / 2];
+}
 
-  for (let i = 0; i < points.length; i++) {
-    d.push(p0[0], p0[1], (p0[0] + p1[0]) / 2, (p0[1] + p1[1]) / 2);
-    p0 = p1;
-    p1 = points[i];
+function getSvgPathFromStroke(points: number[][]): string {
+  if (!points.length) {
+    return "";
   }
 
-  p1 = points[0];
-  d.push(p0[0], p0[1], (p0[0] + p1[0]) / 2, (p0[1] + p1[1]) / 2);
-
-  d.push("Z");
-
-  return d.join(" ");
+  const max = points.length - 1;
+
+  return points
+    .reduce(
+      (acc, point, i, arr) =>
+        i === max ? acc : acc.concat(point, med(point, arr[i + 1])),
+      ["M", points[0], "Q"],
+    )
+    .concat("Z")
+    .join(" ")
+    .replaceAll(/(\s?[A-Z]?,?-?[0-9]*\.[0-9]{0,2})(([0-9]|e|-)*)/g, "$1");
```

Comment:
this regex is a little complex to understand at first glance, can you add a comment here for better readability ?

Reply:
Ok, will add a comment. This regex strips long decimals, so that each point has at max two decimal precision. It's a little easier than using a lot of `Number(point.toFixed(2))`s, I think.